modules/data_team_helper/php/data_team_helper.class.inc

```diff
@@ -398,15 +398,8 @@ class Data_Team_Helper extends \NDB_Form
         }
         //filter for sites
         if ($site!='All Sites') {
-            if ($site == 'All User Sites') {
-                $user           = \User::singleton();
-                $siteID         = $user->getCenterIDs();
-                $where         .= " AND FIND_IN_SET(psc.CenterID, :CID)";
-                $qparams['CID'] = implode(',', $siteID);
-            } else {
                 $where         .= " AND s.CenterID = :CID";
                 $qparams['CID'] = $site;
-            }
```

Comment:
@kongtiaowang I think this check is required to return User sites. Removing this will introduce permission issues.

Reply:
@sruthymathew123 you are right.